### ISSUE TITLE
Simple app and nil pointer dereference

### ISSUE THREAD

**Issue by AnyCPU** [gramework/gramework#23]
Simple app fails in ten hours after starting up with "invalid memory address or nil pointer dereference"

Log
```
   • registering GET /*any
   • Starting HTTP             bind=:80
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x276c08]

goroutine 5933 [running]:
github.com/gramework/gramework.(*node).GetValue(0x442012c360, 0x4422bcc5e0, 0x1a, 0x0, 0x4422b42928, 0x4, 0x20, 0x20)
        /root/go/src/github.com/gramework/gramework/fasthttprouter_tree.go:352 +0x8c8
github.com/gramework/gramework.(*router).Allowed(0x4420095710, 0x4422bcc5e0, 0x1a, 0x4422b42928, 0x4, 0x1a, 0x20)
        /root/go/src/github.com/gramework/gramework/fasthttprouter_router.go:277 +0x27c
github.com/gramework/gramework.(*Router).handle(0x4420086370, 0x4422bcc5e0, 0x1a, 0x4422b42928, 0x4, 0x44200862d0, 0x0, 0x270100, 0x0)
        /root/go/src/github.com/gramework/gramework/router.go:525 +0x268
github.com/gramework/gramework.(*Router).Handler.func1(0x44200862d0)
        /root/go/src/github.com/gramework/gramework/router.go:429 +0x160
github.com/gramework/gramework.(*App).handler.func1(0x4420892d80)
        /root/go/src/github.com/gramework/gramework/app_handler.go:57 +0x1f4
github.com/valyala/fasthttp.(*Server).serveConn(0x44200e6500, 0x3810c0, 0x44209f98e0, 0x101, 0x0)
        /root/go/src/github.com/valyala/fasthttp/server.go:1566 +0x4d8
github.com/valyala/fasthttp.(*Server).(github.com/valyala/fasthttp.serveConn)-fm(0x3810c0, 0x44209f98e0, 0x1, 0x0)
        /root/go/src/github.com/valyala/fasthttp/server.go:1280 +0x30
github.com/valyala/fasthttp.(*workerPool).workerFunc(0x442015a300, 0x4422a8cb80)
        /root/go/src/github.com/valyala/fasthttp/workerpool.go:210 +0x7c
github.com/valyala/fasthttp.(*workerPool).getCh.func1(0x442015a300, 0x4422a8cb80, 0x2a7080, 0x4422a8cb80)
        /root/go/src/github.com/valyala/fasthttp/workerpool.go:182 +0x28
created by github.com/valyala/fasthttp.(*workerPool).getCh
        /root/go/src/github.com/valyala/fasthttp/workerpool.go:181 +0xf8
```

## Expected Behavior
The app outputs the hello world html example.

## Current Behavior
It just fails in some period of time.

## Possible Solution
No idea.

## Steps to Reproduce (for bugs)
1. Compile
2. Either run an executable or run a dockerized executable
3. Wait

## Your Environment
* Go 1.9.* & Go 1.10.*
* Ubuntu 16.04 LTS, Linux the-sun 4.4.114-mainline-rev1 #1 SMP Thu Feb 1 16:23:12 UTC 2018 aarch64 aarch64 aarch64 GNU/Linux
* Standalone app or  Docker 17.12.0-ce

```go
package main

import (
        "fmt"
        "sync"

        g "github.com/gramework/gramework"
)

const (
        appID = "app 0.0.0"
)

func main() {
        lock := &sync.Mutex{}
        n := uint(0)
        app := g.New()
        app.GET("/*any", func(ctx *g.Context) {
                lock.Lock()
                n++
                lock.Unlock()
                txt := fmt.Sprintf("<html><head><title>Welcome</title></head><body><div style=\"text-align:center\">Welcome!</div><br /><div>Views #%v</div></body></html>", n)
                ctx.Response.Header.Set("Server", appID)
                ctx.HTML().WriteString(txt)
        })
        app.ListenAndServe()
}
```

**Comment by kirillDanshin** [comment]
Did you run it both on 1.9 and 1.10rc2?

**Comment by AnyCPU** [comment]
@kirillDanshin, current go version is 'go version go1.10rc2 linux/arm64', previously I have tried Go 1.9.3.

**Comment by kirillDanshin** [comment]
closed by #24

**Comment by kirillDanshin** [comment]
(updated original text to remove reference to issue `#1`)
